public/app/features/alerting/routes.tsx

```diff
@@ -79,10 +79,6 @@ export function getAlertingRoutes(cfg = config): RouteDescriptor[] {
     },
     {
       path: '/alerting/silence/:id/edit',
-      roles: evaluateAccess([
```

Comment:
Decided to not check at the route level (as the permission is dependent on the silence itself). This way, we check once you get to the silence and its loaded - but the user shouldn't be presented with an option to even get there in the UI (all being well)